src/lsstseries/analysis/structure_function/basic/calculator.py

```diff
@@ -0,0 +1,115 @@
+from typing import List
+
+import numpy as np
+from scipy.stats import binned_statistic
+
+from lsstseries.analysis.structure_function.base_argument_container import StructureFunctionArgumentContainer
+from lsstseries.analysis.structure_function.base_calculator import StructureFunctionCalculator
+
+# from lsstseries.analysis.structure_function_argument_containers.structure_function_argument_container import (
+#     StructureFunctionArgumentContainer,
+# )
+# from lsstseries.analysis.structure_function_calculators.structure_function_calculator import (
+#     StructureFunctionCalculator,
+# )
+
+
+class BasicStructureFunctionCalculator(StructureFunctionCalculator):
+    """
+    SF calculation method that calculates excess variance directly as a
+    variance of observations with observational errors subtracted.
+    For reference, please see Equation 12 in https://arxiv.org/abs/1604.05858
+    """
+
+    def __init__(
```

Comment:
I *think* that if all you're doing is calling super(), you don't need to define init in a subclass. Not doing so would reduce the amount of code expected in the calculator subclasses.